src/models/sourcecodemodel.cpp

```diff
@@ -254,8 +254,8 @@ void SourceCodeModel::find(const QString& search, Direction direction, int curre
     auto searchFunc = [&search](const QString& line) { return line.indexOf(search, 0, Qt::CaseInsensitive) != -1; };
 
     auto endReached = [this] { emit searchEndReached(); };
-
-    const int resultIndex = ::search(m_lines, current, direction, searchFunc, endReached);
+    qDebug() << m_lines.mid(m_startLine, m_numLines);
+    const int resultIndex = ::search(m_lines.mid(m_startLine, m_numLines), current, direction, searchFunc, endReached);
```

Comment:
this requires a copy, which is unfortunate... can we pass iterators instead maybe to prevent that?

if only we could use `std::span` already...

Reply:
`midRef`? Otherwise update to c++20?

Reply:
Ah, `midRef` doesn't exists for Lists. And cant :facepalm: